rules/promotions/crowdstrike_external_alerts.toml

```diff
@@ -0,0 +1,106 @@
+min_stack_version = "8.18.0"
+min_stack_comments = "Introduced support for CrowdStrike alert promotion"
+updated_date = "2025/08/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert for each CrowdStrike alert written to the configured indices. Enabling this rule allows you
+to immediately begin investigating CrowdStrike alerts in the app.
+"""
+from = "now-2m"
+index = ["logs-crowdstrike.alert-*"]
+interval = "1m"
+language = "kuery"
+license = "Elastic License v2"
+max_signals = 1000
+name = "CrowdStrike External Alerts"
+note = """## Triage and analysis
+
+### Investigating CrowdStrike External Alerts
+
+CrowdStrike Falcon is a cloud-native endpoint protection platform that delivers real-time threat detection and response capabilities. The 'Behavior - Detected - CrowdStrike Alerts' rule captures security alerts generated by Falcon and enables analysts to investigate threats rapidly based on behavioral indicators and threat intelligence.
+
+### Possible investigation steps
+
+- Review the associated process, file path, and command line to determine whether the activity is legitimate or suspicious.
+- Investigate the user account and host involved in the alert to validate whether the activity was authorized.
+- Cross-reference the alert with CrowdStrike Falcon console for additional context, including process tree, behavioral tags, and threat intelligence matches.
+- Check for any related alerts from the same host, user, or file hash to identify whether this is part of a larger attack chain.
+- Consult the Crowdstrike investigation guide and resources tagged in the alert for specific guidance on handling similar threats.
+
+### False positive analysis
+
+- Alerts involving known and trusted software tools (e.g., remote administration tools) may be false positives. Confirm intent before excluding.
+- Security assessments or penetration testing activities might mimic real threats. Validate the activity with responsible teams.
+- Scheduled jobs, IT scripts, or automation tools may trigger alerts if they behave similarly to malicious code.
+- Review alerts based on detection confidence levels and behavioral scoring to filter out low-confidence or known-benign triggers.
+
+### Response and remediation
+
+- Isolate affected endpoints to prevent lateral movement if malicious behavior is confirmed.
+- Quarantine any identified malicious files and block related hashes or domains.
+- Investigate how the threat entered the environment and close any exploited vulnerabilities.
+- Reset credentials for compromised user accounts or escalate to incident response.
+- Review CrowdStrike Falcon policies and detections to fine-tune future alerting and response coverage.
+- Document the findings and update detection logic or exceptions accordingly.
+"""
+references = ["https://docs.elastic.co/en/integrations/crowdstrike"]
+risk_score = 47
+rule_id = "aeebe561-c338-4118-9924-8cb4e478aa58"
+rule_name_override = "message"
```

Comment:
@peluja1012 the spec on the issue says `Rule name override should be set to rule.name`, but the data and export has `message`